src/ducks/recurrence/RecurrenceBundlePage.jsx

```diff
@@ -0,0 +1,128 @@
+import { useClient, useQuery } from 'cozy-client'
+import React from 'react'
+import { withRouter } from 'react-router'
+import {
+  recurrenceConn,
+  RECURRENCE_DOCTYPE,
+  bundleTransactionsQueryConn
+} from 'doctypes'
+
+import { SubTitle } from 'cozy-ui/transpiled/react/Text'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import Breadcrumbs from 'cozy-ui/transpiled/react/Breadcrumbs'
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+
+import Loading from 'components/Loading'
+import Padded from 'components/Spacing/Padded'
+import {
+  RowDesktop as TransactionRowDesktop,
+  RowMobile as TransactionRowMobile
+} from 'ducks/transactions/TransactionRow'
+import Table from 'components/Table'
+import Header from 'components/Header'
+import BackButton from 'components/BackButton'
+import BarTheme from 'ducks/bar/BarTheme'
+import cx from 'classnames'
+import { prettyLabel } from 'ducks/recurrence/utils'
+// TODO
+import styles from 'ducks/categories/CategoriesHeader.styl'
+import AnalysisTabs from 'ducks/analysis/AnalysisTabs'
+import { BarTitle } from 'components/Title/PageTitle'
+import TransactionsTableHead from 'ducks/transactions/header/TableHead'
+
+const useDocument = (doctype, id) => {
+  const client = useClient()
+  return client.getDocumentFromState(doctype, id)
+}
+
+const BundleInfo = withRouter(({ bundle, router }) => {
+  const { t } = useI18n()
+  const { isMobile } = useBreakpoints()
+  if (!bundle) {
+    return null
+  }
+
+  const goToRecurrenceRoot = () => router.push('/recurrence')
+
+  return (
+    <Header fixed theme="primary">
```

Comment:
No ThemeConsumer?

Reply:
I don't understand the quesiton.

Reply:
I think I miss something. 

You now have introduced a ThemeProvider and a ThemeConsumer, so I was expecting that you define the theme by "screen", and after that access the value (from useTheme or something) on all the below component. Here you still define it manually for one component. There is no utility to define the theme for this page/screen?

Reply:
It's only the Header that has the theme primary.